config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  namespace :api do         
+    namespace :v1 do
+      resources :users
+      post '/login', to: 'users#login'
+      resources :reservations
+      resources :tutors
```

Comment:
I kindly invite you to use `only:` statements for every resource's endpoints here. This could help you close unnecessary endpoints and make your app safer for attacks (and best practices too).

Reply:
Hello @vichuge thank you for the suggestion, its implemented